### PR TITLE
Array rotation in js

### LINKED ISSUE #931: Array_Rotation in javascript
I want to add code for Array_Rotation algorithm in javascript . Please assign me for this.

### PR DESCRIPTION
**Pull request by harshitagupta11**
Fixes #931 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
Array_Rotation algorithm using JS

#### Languages Used:
- Javascript

#### Files Added:
- Array_Rotation.js

### CODE REVIEW

File: Array_Rotation/Array_Rotation.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    temp1 = arr.slice(r, n);
+    //console.log(temp1);
+    temp2 = arr.slice(0, r);
+    //console.log(temp2);
+    var result = []
+    return (result.concat(temp1, temp2));
+}
+
+var arr = [1, 2, 3, 4, 5, 6, 7];
+var r = 2
```
Comment (jainaman224) [inline Array_Rotation/Array_Rotation.js]: Please use better variable name.

File: Array_Rotation/Array_Rotation.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+console.log(arr);
+console.log(Array_Rotation(arr, r, arr.length));
+
+/*
+Input array: {1, 2, 3, 4, 5, 6, 7 }
+Expected Output
+3 4 5 6 7 1 2 
+*/
```
Comment (jainaman224) [inline Array_Rotation/Array_Rotation.js]: Add a single new line at the end of file.

File: Count_Inversion/Count_Inversion.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+function countInversions(arr) {
```
Comment (jainaman224) [inline Count_Inversion/Count_Inversion.js]: Please remove this file as it is not relevant for this PR.

**Comment by amaaniqbal** [comment]
Looks good to me. Well done!

But the new line still seems to be missing at the end.

**Pull request merged**